python/grass/pygrass/modules/grid/grid.py

```diff
@@ -424,6 +424,9 @@ class GridModule(object):
     :type split: bool
     :param mapset_prefix: if specified created mapsets start with this prefix
     :type mapset_prefix: str
+    :param patch_backend: if "r.patch" and overlap == 0, use r.patch, otherwise
+                          use original RasterRow method
```

Comment:
Does not fit with the actual behavior when "r.patch" and `overlap != 0`.

```suggestion
    :param patch_backend: "r.patch", "RasterRow", or None for for default
```

And later describe in two or three sentences like Python documentation usually does:

When *patch_backend* is None, the RasterRow method is used for patching the result. When *patch_backend* is "r.patch", r.patch is used. r.patch can only be used when *overlap* is 0.

Reply:
Maybe also say that r.patch is used _with parallelization_.